treeherder/intermittents_commenter/commenter.py

```diff
@@ -353,7 +353,7 @@ def get_test_variant(self, test_suite):
         base_symbol = test_suite
         found_variants = []
         for variant in variant_symbols:
-            if f"-{variant}" in base_symbol:
+            if f"-{variant}-" in base_symbol:
```

Comment:
there are some variants that might not have a `-<variant>-` pattern, such as [eme](https://treeherder.mozilla.org/jobs?repo=mozilla-central&searchStr=test-%2Ceme&selectedTaskRun=ImO_Yn_URrWBIdz58uDdbw.0).

maybe it could be:
`if f"-{variant}-" in base_symbol or base_symbol.endswith(variant):`